API/paypal/index.js

```diff
@@ -31,7 +31,7 @@ module.exports = ({ config, db }) => {
     try {
       order = await paypalClient.client(config.extensions.paypal).execute(request);
     } catch (err) {
-      console.error(err);
+      console.error('Error on the complete PayPal order function. Please check the error, the request of the body and of the order:', err, req.body, request);
```

Comment:
`Error on the complete PayPal order function.` -> `Error on the complete PayPal order.`
@rozzilla Pls, fix text